bayem/checks.py

```diff
@@ -0,0 +1,78 @@
+from collections import defaultdict
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from .vba import DictModelError
+
+
+def _measure(true, lin, norm):
+    a, b = np.asarray(true), np.asarray(lin)
+    error = norm(a - b)
+    ref = norm(b)
```

Comment:
check for norm==0??

Reply:
By default, python would give a RuntimeWarning and return NaN, if ref==0. I think that is fine. 
However, to show that we thought about it, we now handle that case explicitly and explain it a bit 
https://github.com/BAMresearch/bayem/blob/585a2716215d551a54de83c4d20f9a552bff6287/bayem/checks.py#L13-L20